setuptools/tests/test_namespaces.py

```diff
@@ -12,10 +12,14 @@
 
 class TestNamespaces:
 
-    @pytest.mark.xfail(sys.version_info < (3, 5),
-        reason="Requires importlib.util.module_from_spec")
-    @pytest.mark.skipif(bool(os.environ.get("APPVEYOR")),
-        reason="https://github.com/pypa/setuptools/issues/851")
+    @pytest.mark.xfail(
+        sys.version_info < (3, 5),
+        reason="Requires importlib.util.module_from_spec",
+    )
+    @pytest.mark.xfail(
+        os.environ.get("APPVEYOR"),
+        reason="https://github.com/pypa/setuptools/issues/851",
+    )
```

Comment:
I think you can drop the APPVEYOR `xfail` entirely, since it seems that this test is only failing on Appveyor for Python 2.7, which is covered by the `xfail` anyway.

Reply:
You are right, removed.